neo/Consensus/ConsensusService.cs

```diff
@@ -221,7 +296,11 @@ private void OnConsensusPayload(ConsensusPayload payload)
                 case ConsensusMessageType.PrepareResponse:
                     OnPrepareResponseReceived(payload, (PrepareResponse)message);
                     break;
+                case ConsensusMessageType.CommitAgreement:
+                    OnCommitAgreement(payload, (CommitAgreement)message);
+                    break;
             }
+            Log($"OMG! WE DID IT :)");
```

Comment:
Please remove the funny code :P

Reply:
EHuaheauea, the happiness of the achievement. jajajajja
We gonna debug a little more and then clean everything, garbage collector :dancer: